### PR TITLE
removed references to Pongo, said that models are in progress

### PR DESCRIPTION
**Pull request by andrewkern**
hey @agladstein-- can you look over my shoulder on this PR? trying to erase mention of Pongo...

### CODE REVIEW

**Comment by andrewkern** [comment]
this should close https://github.com/popsim-consortium/manuscript/issues/104

**Comment by agladstein** [comment]
Will do. But, I'm going offline for a few hrs.

File: popSimManu.tex [hunk 1]
```diff
@@ -221,7 +221,7 @@ \section*{Introduction}
 This resource, which we call \stdpopsim, makes running
 realistic simulations for population genetic analysis a simple matter of
 choosing pre-implemented models from a community-maintained catalog.
-The \stdpopsim catalog currently contains five organisms: humans, \emph{Pongo~pygmaeus},
+The \stdpopsim catalog currently contains four organisms: humans,
```
Comment (jradrion) [inline popSimManu.tex]: Currently Figure 1 and Table 1 only have 3 taxa.

Comment (andrewkern) [reply popSimManu.tex]: okay i can make it just 3 for the paper

**Review by jradrion — APPROVED** [review]
Looks good!

**Comment by agladstein** [comment]
late to the party. but, looks good.

**Pull request merged**